api/v1/endpoints/queries.sql

```diff
@@ -83,14 +83,18 @@ WITH object_data AS (
     SELECT
         a.id AS assessment_id,
         a.tid AS assessment_name,
-        a.title as assessment_title,
+        a.title AS assessment_title,
         aar.credit,
         aar.end_date,
-        aar.id as id,
+        aar.exam_uuid,
+        aar.id AS assessment_access_rule_id,
         aar.mode,
         aar.number,
```

Comment:
```suggestion
        aar.number AS assessment_access_rule_number,
```

We generally rename `id` and `number` columns to table-specific versions, because many tables share the same column names.